pkg/vtpm/Dockerfile

```
@@ -19,6 +19,7 @@ RUN ./autogen.sh --prefix=/usr --with-tpm2
 RUN make -j$(nproc)
 RUN make -j$(nproc) install
 RUN cp /usr/lib/libtpms.so.* /out/usr/lib/
+RUN strip /out/usr/lib/libtpms.so.*
```

```suggestion
RUN strip --strip-unneeded /out/usr/lib/libtpms.so.*
```

as of https://www.linuxfromscratch.org/lfs/view/development/chapter08/stripping.html

---

@christoph-zededa this is a default behavior but I'll commit it to be on a safe side